contracts/ocr2/src/integration_tests.rs

```diff
@@ -801,3 +801,68 @@ fn set_link_token() {
         .unwrap();
     assert_eq!(token_addr, new_link_token);
 }
+
+#[test]
+fn set_billing_payout() {
+    let mut env = setup();
+    let deposit = Decimal::from_str("1000").unwrap().0;
```

Comment:
Seems unused in the test.

Reply:
resolved the changes during the rebase fixes